Buildings/Experimental/DHC/Loads/Steam/BaseClasses/Examples/ValveSelfActing.mo

```diff
@@ -0,0 +1,85 @@
+within Buildings.Experimental.DHC.Loads.Steam.BaseClasses.Examples;
+model ValveSelfActing
+  "Example model for self-acting steam valve with a noisy inlet pressure input"
+  extends Modelica.Icons.Example;
+  package MediumSte = Buildings.Media.Steam "Medium model";
+  parameter Modelica.Units.SI.MassFlowRate m_flow_nominal=1
+    "Nominal mass flow rate";
+
+  Buildings.Experimental.DHC.Loads.Steam.BaseClasses.ValveSelfActing prv(
+    redeclare package Medium = MediumSte,
```

Comment:
```suggestion
    redeclare final package Medium = MediumSte,
```

(and all the other assignment later)

Reply:
Since this is an example model, I believe `final` may not be required.  
Please let me know what you think.

Reply:
You are right that it is not required, whether in a model or in an example, and if you do a broad search on the Buildings library, you'll see about an equal amount of `redeclare package Medium = Medium` and `redeclare final package Medium = Medium`.
But I personally prefer to make those redeclarations final since there should not be any conceivable cases where a user would extend or use these models and cutomize the underlying component with a different Medium. Even though it is an example, a user might want to extend it to test other parameters value, in which case there shouldn't be an option to change the propagation of the medium from the top-level class down to the components.

Does that make sense?

Reply:
Thanks for explaining!
I have added `final` to medium declarations in the example.